components/director/internal/domain/api/converter_test.go

```diff
@@ -160,6 +189,11 @@ func TestConverter_InputFromGraphQL(t *testing.T) {
 			FetchRequestConverter: func() *automock.FetchRequestConverter {
 				return &automock.FetchRequestConverter{}
 			},
+			VersionConverter: func() *automock.VersionConverter {
+				conv := &automock.VersionConverter{}
+				conv.On("InputFromGraphQL", emptyGQLAPIDefinition.Version).Return(nil).Once()
+				return conv
+			},
 		},
```

Comment:
Missing case when input is nil.

Reply:
done

Reply:
Looks like that case still isn't covered by tests both in `api/converter_test.go` and `eventapi/converter_test.go`.